projects/packages/connection/src/sso/class-user-admin.php

```diff
@@ -0,0 +1,1292 @@
+					$event,
+					array(
+						'success'       => 'false',
+						'error_message' => $message_type,
+					)
+				);
+				return self::create_error_notice_and_redirect( $query_params );
+			}
+
+			$body                    = json_decode( $response['body'] );
+			$invite_response_message = $body->success ? 'reinvited-success' : 'failed';
+			$query_params            = array(
+				'jetpack-sso-invite-user' => $invite_response_message,
+				'_wpnonce'                => $nonce,
+			);
+
+			if ( ! $body->success ) {
+				self::$tracking->record_user_event(
+					$event,
+					array(
+						'success'       => 'false',
+						'error_message' => $invite_response_message,
+					)
+				);
+			} else {
+				self::$tracking->record_user_event( $event, array( 'success' => 'true' ) );
+			}
+
+			return self::create_error_notice_and_redirect( $query_params );
+		} else {
+			$error        = 'empty-invite';
+			$query_params = array(
+				'jetpack-sso-invite-user'  => 'failed',
+				'jetpack-sso-invite-error' => 'empty-invite',
+				'_wpnonce'                 => $nonce,
+			);
+			self::$tracking->record_user_event(
+				$event,
+				array(
+					'success'       => 'false',
+					'error_message' => $error,
+				)
+			);
+			return self::create_error_notice_and_redirect( $query_params );
+		}
+	}
+
+	/**
+	 * Adds 'Revoke invite' and 'Resend invite' link to user table row actions.
+	 * Removes 'Reset password' link.
+	 *
+	 * @param array   $actions - User row actions.
+	 * @param WP_User $user_object - User object.
+	 */
+	public function jetpack_user_table_row_actions( $actions, $user_object ) {
+		$user_id            = $user_object->ID;
+		$has_pending_invite = self::has_pending_wpcom_invite( $user_id );
+
+		if ( current_user_can( 'promote_users' ) && $has_pending_invite ) {
+			$nonce                        = wp_create_nonce( 'jetpack-sso-revoke-user-invite' );
+			$actions['sso_revoke_invite'] = sprintf(
+				'<a class="jetpack-sso-revoke-invite-action" href="%s">%s</a>',
+				add_query_arg(
+					array(
+						'action'              => 'jetpack_revoke_invite_user_to_wpcom',
+						'user_id'             => $user_id,
+						'revoke_invite_nonce' => $nonce,
+						'invite_id'           => $has_pending_invite,
+					),
+					admin_url( 'admin-post.php' )
+				),
+				esc_html__( 'Revoke invite', 'jetpack-connection' )
+			);
+		}
+		if ( current_user_can( 'promote_users' ) && $has_pending_invite ) {
+			$nonce                        = wp_create_nonce( 'jetpack-sso-resend-user-invite' );
+			$actions['sso_resend_invite'] = sprintf(
+				'<a class="jetpack-sso-resend-invite-action" href="%s">%s</a>',
+				add_query_arg(
+					array(
+						'action'              => 'jetpack_resend_invite_user_to_wpcom',
+						'user_id'             => $user_id,
+						'resend_invite_nonce' => $nonce,
+						'invite_id'           => $has_pending_invite,
+					),
+					admin_url( 'admin-post.php' )
+				),
+				esc_html__( 'Resend invite', 'jetpack-connection' )
```

Comment:
Since we're changing the translation domain, is there a way to somehow copy translations from `jetpack` to `jetpack-connection` so they wouldn't need to be translated again?

Reply:
I believe that's going to be handled automatically on release. That's not something we can handle right here afaik.